tests/unit/materializers/test_mlx_array_materializer.py

```diff
@@ -0,0 +1,30 @@
+#  Copyright (c) ZenML GmbH 2025. All Rights Reserved.
```

Comment:
One more thing, can you move this to `tests/integration/integrations/mlx` instead, as I think the environment in which our unit tests run don't usually have all the integrations installed.